src/playwright_ui5_select/__init__.py

```diff
@@ -27,7 +27,13 @@ def _export_wrap(raw: str) -> str:
     Wraps the raw content in a manner that allows it to be executed in a browser context.
     See https://github.com/microsoft/playwright/issues/16705 for details.
```

Comment:
Let's update the comment as well to the latest post.
```suggestion
    See https://github.com/microsoft/playwright/issues/36448 for details.
```